ARSIS-Telemetry-GroundControl/LMCC/src/routes/hololenstest/+page.svelte

```diff
@@ -0,0 +1,12 @@
+<script>
+	import { Input, Video } from 'flowbite-svelte';
+
+	let ipaddr = '192.168.125.50';
+</script>
+
+<Input bind:value={ipaddr} />
+<Video
+	src="http://{ipaddr}/api/holographic/stream/live_high.mp4"
```

Comment:
Video doesn't seem to reload when the `src` gets changed. I had to manually invoke `load()` and `play()` on the video element in order to see the feed.